### DISCUSSION TITLE
Feature Request: People with only VIEW rights should be able to copy a 'read-only' panel.

### DISCUSSION THREAD

**Discussion by j-dc** [grafana/grafana#69785] [Dashboards, Panels & Visualizations]
I have users who have only view rights on certain folders, and admin rights on other folders.

They are not able to copy a panel from the 'view only' dashboard to paste on their 'edit' dashboards.

The option 'copy' should always be available to users who have at least one dashboard where they can paste panels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82655. Feel free to continue the discussion around this there. Thank you!
